units/patch_intent.go

```diff
@@ -356,14 +356,14 @@ func (j *patchIntentProcessor) finishPatch(ctx context.Context, patchDoc *patch.
 	}
```

Comment:
I'm not sure I understand why it is that the module needs to be specified as Evergreen in Spruce's case. Since the patch is on the evergreen project, shouldn't that happen automatically?

Reply:
Discussed over slack. I understand it better now and will think about how to add more help text to help the user understand as well. 
I'll review again once [this](https://github.com/evergreen-ci/evergreen/pull/5512#issuecomment-1076414623) is addressed.

Reply:
Sorry, [the comment you're referring to](https://github.com/evergreen-ci/evergreen/pull/5512#issuecomment-1076414623) was unclear. What I meant was that the fix already in this PR should also address the issue for PRs as well. I was coming to say that [my original comment](https://github.com/evergreen-ci/evergreen/pull/5512#issue-1177376627) implied the only issue was for CLI patches but I realized it was also an issue for PR patches.

Would you mind to review again at your earliest convenience?

Reply:
ah that makes sense, thanks!